Thanks for the PR @Kanwaljeet1! 
However there is a TypeScript compilation error in src/lib/store.ts on line 407:
error TS18004: No value exists in scope for the shorthand property 'jobs'
Please add this line in the useJobApplications hook before returning:
const jobs = jobsQuery.data ?? [];
Let me know once fixed!

--- a/src/lib/store.ts
+++ b/src/lib/store.ts
@@ -404,6 +404,7 @@ export function useJobApplications(userId: string | undefined) {
     return deleteJobMutation.mutateAsync(id);
   }, [deleteJobMutation]);
 
+  const jobs = jobsQuery.data ?? [];
   return { jobs, addJob, updateJob, deleteJob, jobsError: jobsQuery.error instanceof Error ? jobsQuery.error.message : null, jobsLoading: jobsQuery.isLoading };
 }
 

